waku/node/v2/waku_message.nim

```diff
@@ -0,0 +1,4 @@
+type
```

Comment:
Belongs in waku_types along with all the other types that are at the bottom, don't really see a good reason to have a new file for this

Reply:
Because we have the message subscription filter which is used in types which is then needed in waku_node which then needs waku_message which is in waku_types causing an import cycle.

Reply:
So you move the MessageNotificationSubscriptionsType(s) to `waku_types`? I don't see the issue.

Reply:
Thats what that module is for

Reply:
That module is gonna get very cluttered if we do that, but sure.